sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -458,6 +458,7 @@ private void removeAllStoredErrorLogFiles(UUID id) {
     private void removeStoredThrowable(UUID id) {
         mErrorReportCache.remove(id);
         ErrorLogHelper.removeStoredThrowableFile(id);
+        WrapperSdkExceptionManager.deleteWrapperExceptionData(id);
```

Comment:
Maybe what makes sense is to null check the id above the call to deleteWrapperExceptionData and leave the rest as is?